data/school-types.csv

```diff
@@ -0,0 +1,72 @@
+col_name,breakdown_topic,filter_item,filter_grouping_column,filter_item_grouping,school_census_code
+establishment_type_group,Establishment type group,Academy,,,
+establishment_type_group,Establishment type group,Total,,,
+establishment_type_group,Establishment type group,All state funded,,,
+establishment_type_group,Establishment type group,Independent,,,
+establishment_type_group,Establishment type group,LA maintained,,,
```

Comment:
Did you consider using 'Local authority maintained' instead of just LA? (I think we can argue it either way)

Reply:
LA maintained is just the current school census standard. This could be a good chance to push towards avoiding the acronym to be fair.

Reply:
Might as well take the chance to push for it then! Do you want to do that before merging this, or merge this and then push in parallel and update here if we change it later?

Reply:
I've updated away from he acronym here and on the guidance. Ann didn't have any problem with expanding it out.